deployer/pipe-listener.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+# shellcheck disable=SC2046
+while true; do (cd /var/docker/setup; ./interactive_setup.sh $(cat deployer/arg-pipe) &> deployer/deploy-log.txt); done
```

Comment:
I am wondering if this script would be logically better kept in the deployer-backend project. But in the end it depends on both ndb-setup and deployer-backend of course, so I see why you have it here (also for easier execution)

Reply:
As I think it has a much higher dependency on the `interactive-setup.sh` script, I prefer to have it here. In the end this is kind of a usage and other systems could also use this service with a different approach.